kubernetes-crds/pkg/client/clientset/versioned/fake/register.go

```diff
@@ -29,7 +29,7 @@ var codecs = serializer.NewCodecFactory(scheme)
 var parameterCodec = runtime.NewParameterCodec(scheme)
 
 func init() {
-	v1.AddToGroupVersion(scheme, schema.GroupVersion{Version: "v1alpha1"})
+	v1.AddToGroupVersion(scheme, schema.GroupVersion{Version: "v1"})
```

Comment:
why has this (and below) changed to "v1"?

Reply:
After second inspection, there is nothing here that should refer to these CRDs (riff's) version. So maybe it was a bug in the generator previously (or we changed too much by hand)